skiko/src/commonMain/kotlin/org/jetbrains/skia/TextBlob.kt

```diff
@@ -319,6 +319,59 @@ class TextBlob internal constructor(ptr: NativePointer) : Managed(ptr, _Finalize
     private object _FinalizerHolder {
         val PTR = TextBlob_nGetFinalizer()
     }
+
+    override fun iterator(): Iterator<TextBlobIter.Run> = TextBlobIter(this)
+
+    class TextBlobIter(val textBlob: TextBlob) : Iterator<TextBlobIter.Run>, Managed(Iter_nCreate(textBlob._ptr), FinalizerHolder.PTR) {
+        data class Run(val typeface: Typeface, val glyphs: ShortArray) {
+            override fun equals(other: Any?): Boolean {
+                if (this === other) return true
+                if (other == null || this::class != other::class) return false
+
+                other as Run
+
+                if (typeface != other.typeface) return false
+                if (!glyphs.contentEquals(other.glyphs)) return false
```

Comment:
Maybe just `return glyphs.contentEquals(other.glyphs)`?

Reply:
OK, this is autogenerated by idea )